projects/schematics/src/migrations/2_0/ng-add-localize/ng-add-localize.ts

```diff
@@ -45,12 +46,34 @@ function installPackageJsonDependencies(): Rule {
   };
 }
 
+function isAngularLocalizeInstalled(tree: Tree): boolean {
+  const pkgPath = '/package.json';
+  const buffer = tree.read(pkgPath);
+  if (!buffer) {
+    throw new SchematicsException('Could not find package.json');
+  }
+
+  return buffer.toString().includes(ANGULAR_LOCALIZE);
```

Comment:
I would feel safer if we check strictly into (dev)dependencies, instead of overall inclusion